src/__test__/Home.test.jsx

```diff
@@ -0,0 +1,49 @@
+import {
```

Comment:
- [ ] As the previous reviewer suggested kindly make sure your tests are passing as for now they are not passing
![Screenshot_20230106_175031](https://user-images.githubusercontent.com/25227022/211058565-78aea25b-88f1-4d0a-810a-d01736582c32.png)